google/cloud/aiplatform/featurestore/entity_type.py

```diff
@@ -381,3 +386,836 @@ def delete_features(self, feature_ids: List[str], sync: bool = True,) -> None:
 
         for feature in features:
             feature.wait()
+
+    @base.optional_sync()
+    def delete(self, force: bool = False, sync: bool = True) -> None:
+        """Deletes this EntityType resource. If force is set to True,
+        all features in this EntityType will be deleted prior to entityType deletion.
+
+        WARNING: This deletion is permanent.
+
+        Args:
+            force (bool):
+                If set to true, any Features for this
+                EntityType will also be deleted.
+                (Otherwise, the request will only work
+                if the EntityType has no Features.)
+            sync (bool):
+                Whether to execute this deletion synchronously. If False, this method
+                will be executed in concurrent Future and any downstream object will
+                be immediately returned and synced when the Future has completed.
+        Raises:
+            FailedPrecondition: If features are created in this EntityType and force = False.
+        """
+        _LOGGER.log_action_start_against_resource("Deleting", "", self)
+        lro = getattr(self.api_client, self._delete_method)(
+            name=self.resource_name, force=force
+        )
+        _LOGGER.log_action_started_against_resource_with_lro(
+            "Delete", "", self.__class__, lro
+        )
+        lro.result()
+        _LOGGER.log_action_completed_against_resource("deleted.", "", self)
+
+    @classmethod
+    @base.optional_sync()
+    def create(
+        cls,
+        entity_type_id: str,
+        featurestore_name: str,
+        description: Optional[str] = None,
+        labels: Optional[Dict[str, str]] = None,
+        project: Optional[str] = None,
+        location: Optional[str] = None,
+        credentials: Optional[auth_credentials.Credentials] = None,
+        request_metadata: Optional[Sequence[Tuple[str, str]]] = (),
+        sync: bool = True,
+    ) -> "EntityType":
+        """Creates an EntityType resource in a Featurestore.
+
+        Example Usage:
+
+            my_entity_type = aiplatform.EntityType.create(
+                entity_type_id='my_entity_type_id',
+                featurestore_name='projects/123/locations/us-central1/featurestores/my_featurestore_id'
+            )
+            or
+            my_entity_type = aiplatform.EntityType.create(
+                entity_type_id='my_entity_type_id',
+                featurestore_name='my_featurestore_id',
+            )
+
+        Args:
+            entity_type_id (str):
+                Required. The ID to use for the EntityType, which will
+                become the final component of the EntityType's resource
+                name.
+
+                This value may be up to 60 characters, and valid characters
+                are ``[a-z0-9_]``. The first character cannot be a number.
+
+                The value must be unique within a featurestore.
+            featurestore_name (str):
```

Comment:
nit. Should we mentioned that featurestore (or parent resource in general) should exist before this call? Same for other create function.